packages/compass-e2e-tests/helpers/commands/get-connect-form-state.ts

```diff
@@ -313,6 +334,19 @@ async function getValue(
   return value || null;
 }
 
+async function getAceValue(
```

Comment:
silly nit, would it be better to name this `getEditorValue`, anticipating us replacing Ace eventually?

Reply:
I made it consistent with `browser.setAceValue()` that is already in use.